### PR TITLE
tls: Remove tls certificates from reonciled secret (PROJQUAY-2606)

### PR DESCRIPTION
**Pull request by jonathankingfc**
- Prevent old tls certs from being included in new secret generated by controller

### CODE REVIEW

File: pkg/configure/configure.go [hunk 1]
```diff
@@ -182,6 +182,9 @@ func createUpdatedSecret(reconfigureRequest request, oldSecret corev1.Secret) co
 	}
 
 	for prop, value := range oldSecret.Data {
+		if prop == "ssl.key" || prop == "ssl.cert" {
+			continue
+		}
```
Comment (ricardomaraschini) [inline pkg/configure/configure.go]: I don't think we need this `oldSecret` function argument anymore. I remember introducing this to solve https://issues.redhat.com/browse/PROJQUAY-2419 but it seems like after your fixes in the config editor it works without it. Can you remove it?

**Review by ricardomaraschini — CHANGES_REQUESTED** [review]

File: pkg/configure/configure.go (hunk 1 above)
Comment (ricardomaraschini) [reply pkg/configure/configure.go]: i.e. prior to your fix in the config editor it was not sending ssl.cert and ssl.key, hence this hack to inherit whatever was in the old secret.

Comment (jonathankingfc) [reply pkg/configure/configure.go]: Removed the logic for the old secret

**Review by ricardomaraschini — APPROVED** [review]

**Comment by ricardomaraschini** [comment]
@jonathankingfc don't forget to cherry pick this into 3.6

**Pull request merged**